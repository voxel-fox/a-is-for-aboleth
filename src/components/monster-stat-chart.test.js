import StatChart from "./monster-stat-list";

describe("<StatChart />", () => {
  const data = [
    {
      attr: "Strength",
      label: "STR",
      value: 11
    },
    {
      attr: "Dexterity",
      label: "DEX",
      value: 13
    }
  ];

  it("renders correctly", () => {
    const wrapper = shallow(<StatChart data={data} />);
    expect(wrapper).toMatchSnapshot();
  });
});
