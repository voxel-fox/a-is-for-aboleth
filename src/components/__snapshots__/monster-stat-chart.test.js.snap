// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StatChart /> renders correctly 1`] = `
<dl
  css={
    Object {
      "name": "1m0avwd-statListStyle",
      "styles": "display:flex;width:300px;justify-content:center;label:statListStyle;",
    }
  }
>
  <StatRow
    key="STR"
    stat={
      Object {
        "attr": "Strength",
        "label": "STR",
        "value": 11,
      }
    }
  />
  <StatRow
    key="DEX"
    stat={
      Object {
        "attr": "Dexterity",
        "label": "DEX",
        "value": 13,
      }
    }
  />
</dl>
`;
